pkgs/by-name/uh/uhhyou-plugins-juce/package.nix

```diff
@@ -0,0 +1,69 @@
+{
+  alsa-lib,
+  cmake,
+  curl,
+  fetchFromGitHub,
+  freetype,
+  lib,
+  libGL,
+  libXcursor,
+  libXext,
+  libXinerama,
+  libXrandr,
+  libjack2,
+  pkg-config,
+  stdenv,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "uhhyou-plugins-juce";
+  version = "0.1.0";
+  src = fetchFromGitHub {
+    owner = "ryukau";
+    repo = "UhhyouPluginsJuce";
+    tag = "UhhyouPluginsJuce${finalAttrs.version}";
+    hash = "sha256-oHxyOTiqEwdNUKGQNjjfdRkzMa+4TYX6Vf6ZS9BTcC0=";
+    fetchSubmodules = true;
+  };
+  nativeBuildInputs = [
+    pkg-config
+    cmake
+  ];
+  buildInputs = [
+    alsa-lib
+    curl
+    freetype
+    libGL
+    libXcursor
+    libXext
+    libXinerama
+    libXrandr
+    libjack2
+  ];
+
+  # Disable LTO to avoid optimization mismatch issues
+  NIX_CFLAGS_COMPILE = [
+    "-fno-lto"
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/lib/vst3
+
+    for plugin in TwoBandStereo SlopeFilter EasyOverdrive AmplitudeModulator; do
+      if [ -d ''${plugin}_artefacts/Release/VST3/''${plugin}.vst3 ]; then
```

Comment:
Another small note but irrelevant if you'd apply [the previous comment](https://github.com/NixOS/nixpkgs/pull/456010/files#r2469381424), no need to use the Nix specific syntax of `''${var}`, use simply `$var`.